multimodal/src/autogluon/multimodal/models/mmocr_text_detection.py

```diff
@@ -0,0 +1,173 @@
+import logging
+import warnings
+from typing import Optional
+
+import torch
+from mim.commands.download import download
+from torch import nn
+
+try:
+    import mmcv
+    from mmcv.parallel import scatter
+    from mmcv.runner import load_checkpoint
+except ImportError:
+    mmcv = None
+
+try:
+    import mmocr
+    from mmocr.models import build_detector
+    from mmocr.utils.model import revert_sync_batchnorm
+except ImportError:
+    mmocr = None
+
+try:
+    import mmdet
+    from mmdet.core import get_classes
+except ImportError:
+    mmdet = None
+
+from ..constants import AUTOMM, BBOX, COLUMN, COLUMN_FEATURES, FEATURES, IMAGE, IMAGE_VALID_NUM, LABEL, LOGITS, MASKS
+from .utils import assign_layer_ids, get_column_features, get_model_head
+
+logger = logging.getLogger(AUTOMM)
+
+
+class MMOCRAutoModelForTextDetection(nn.Module):
+    """
+    Support MMOCR object detection models.
+    Refer to https://github.com/open-mmlab/mmdetection
```

Comment:
The link is wrong, please link it to mmocr repo.

Reply:
Sorry, I will modify it.